pallets/keystore/src/lib.rs

```diff
@@ -129,11 +129,20 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub (super) fn deposit_event)]
 	pub enum Event<T: Config> {
 		/// A key was added.
-		KeyAdded(T::AccountId, T::Hash, KeyPurpose, KeyType),
+		KeyAdded {
+			account_id: T::AccountId,
```

Comment:
maybe we can name this more meaningfully? I assume the account here is the key `owner`?

Reply:
No idea. Maybe someone else can confirm it?. To be honest, I wrote `account_id` because it was the name it was referring to when the previous enum was deconstructed.

Reply:
cc @cdamian

Reply:
`owner` makes sense. 

To give some more context - these keys are generated by the p2p node and then added for every account that is created on the node.

Reply:
Maybe @offerijns can confirm what Nuno's say? 😃